react/Field/index.jsx

```diff
@@ -51,6 +53,18 @@ class InputPassword extends React.Component {
             {visible ? hideLabel : showLabel}
           </div>
         )}
+        {showVisibilityButton && secondaryComponent && (
```

Comment:
Nit: I think the naming is not optimal here : showSecondaryComponent ou visibilityComponent mais pas les deux ?